workflow-bot-app/src/main/java/com/symphony/bdk/workflow/converter/BiConverter.java

```diff
@@ -0,0 +1,37 @@
+package com.symphony.bdk.workflow.converter;
+
+import java.lang.reflect.ParameterizedType;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.BiFunction;
+import java.util.stream.Collectors;
+
+public interface BiConverter<S, K, T> extends BiFunction<S, K, T> {
+  /**
+   * The source type for which this is going to convert.
+   *
+   * @return the source class type
+   */
```

Comment:
is it safe to use hardcoded indexes? 0 and 2 ?

Reply:
yes, safe